src/Symfony/Installer/DemoCommand.php

```diff
@@ -38,22 +38,29 @@ protected function initialize(InputInterface $input, OutputInterface $output)
     {
         parent::initialize($input, $output);
 
-        $this->projectDir = getcwd();
+        if ('symfony_demo' === $input->getArgument('directory')) {
```

Comment:
I would do this only if the argument has not been given at all, but not depending on the vaue if given.